File: Card.java
public class Card{
    public int value;
    public String suit;
    public int getValue(){
        return this.value;
    }
    public String getSuit(){
        return this.suit;
    }
    public Card(int value, String suit){
        this.value = value;
        this.suit = suit;
    }

    public String toString()
    {   
       String valueString = value+"";
       if(value==11){
        valueString = "Jack";
       }
       else if(value==12){
       valueString = "Queen";
       }
       else if(value == 13){
        valueString = "King";
       }
       else if(value == 1){
        valueString = "Ace";
       }
       return valueString;
    }
}

File: card.py
class card:
    def __init__(self,rank,suit):
        self.rank = rank
        self.suit = suit
    def string(self):
        name = ""
        if self.rank == 13:
            name += "King"
        elif self.rank == 12:
            name += "Queen"
        elif self.rank == 11:
            name += "Jack"
        elif self.rank == 1:
            name += "Ace"
        else:
            name += str(self.rank)
        name += " of "+self.suit
        return name